### DISCUSSION TITLE
Can you expand on the documentation for manual steps

### DISCUSSION THREAD

**Discussion by steowens** [NitorCreations/nflow#463] [Q&A] [answered]
There does not seem to be any examples or discussion on manual steps.  A common use case is start a workflow, update some state variables, shift to a manual state which awaits input from a user.  Such as an access request. / approval workflow.  The approver on receiving a notification logs in to some UI and a Rest API is invoked which displays the request to the approver, the approver then submits a request to the Rest API (via a web form) and the workflow responds to this request (somehow but currently how is a complete mystery) and either grants access or notifies the requester that access was denied.

**Comment by efonsell** [comment, marked as answer]
There was some discussion here: https://groups.google.com/g/nflow-users/c/IGnCZR33jZI

I also updated the workflow instance state descriptions a bit here: https://github.com/NitorCreations/nflow/wiki/Glossary#state

Was there anything else that was unclear related to this?

**Reply by steowens** [reply]
It might be helpful in the discussion of manual states to provide some code examples of how to manually change the state.  Other than that the state descriptions are looking pretty good.

**Reply by efonsell** [reply]
Added this entry to FAQ: https://github.com/NitorCreations/nflow/wiki/FAQ#how-to-move-a-workflow-instance-from-a-manual-state-to-some-other-state-eg-after-the-manual-processing-has-been-done
